src/com/magento/idea/magento2plugin/magento/packages/eav/AttributeProperty.java

```diff
@@ -22,7 +22,10 @@
     VISIBLE_ON_FRONT("visible_on_front"),
     APPLY_TO("apply_to"),
     OPTION("option"),
-    BACKEND_MODEL("backend");
+    BACKEND_MODEL("backend"),
+    USER_DEFINED("user_defined"),
+    POSITION("position"),
+    SYSTEM("system");
```

Comment:
I don't think that it makes sense to give the possibility to configure both `user_defined` and `system`. Since `user_defined = true`, automatically means `system = false`, and vice versa.